app/models/subnet.rb

```diff
@@ -58,6 +59,12 @@ def model_name
     :api_description => N_('HTTPBoot Proxy ID to use within this subnet'),
     :description => N_('HTTPBoot Proxy to use within this subnet')
 
+  belongs_to_proxy :external_ipam,
+    :feature => N_('external_ipam'),
+    :label => N_('IP Address Management with various external IPAM providers'),
+    :api_description => N_('IPAM Proxy ID to use within this subnet'),
+    :description => N_('IPAM Proxy to use within this subnet')
```

Comment:
I suggest to name the feature as `External IPAM`.

Reply:
How do I best handle feature names with spaces? In the Smart Proxy plugin, I have changed the plugin name as follows: 

```
  class Plugin < ::Proxy::Plugin
    plugin 'External IPAM', Proxy::Ipam::VERSION

    http_rackup_path File.expand_path('ipam_http_config.ru', File.expand_path('../', __FILE__))
    https_rackup_path File.expand_path('ipam_http_config.ru', File.expand_path('../', __FILE__))
  end
```

The Smart Proxy does not pick this up when there is a space in the name for some reason. 

Should I add an underscore(instead of space) in the plugin name, then handle in the `associate_features` of the smart_proxy model so that feature names can be mapped?

Reply:
Oh right, then I suggest to use only "IPAM" for the Smart Proxy API and "External IPAM" for the Foreman core IPAM provider.

Reply:
@lzap I'm having a tough time getting this to work using "IPAM" for the Smart Proxy API and "External IPAM" for the Foreman core IPAM provider. Can we name it `IPAM` in both places?